### PR TITLE
TCVP-2872: Add Upload Documents Feature to Ticket Validation Screen

### PR DESCRIPTION
**Pull request by saikrishnametpalli-nttdata**
# Description

This PR includes the following proposed change(s):

- [TCVP-2872](https://justice.gov.bc.ca/jira/browse/TCVP-2872)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by KDNeufeld — APPROVED** [review]
LGTM

File: src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.html [hunk 1]
```diff
@@ -373,6 +373,8 @@ <h4>Who are we contacting?</h4>
       </mat-expansion-panel>
       <app-ticket-request [disputeInfo]="lastUpdatedDispute"
         *ngIf="lastUpdatedDispute.emailAddressVerified === true || !(lastUpdatedDispute.emailAddress)"></app-ticket-request>
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.html:375]: You can use ng-container to include these three code blocks, so you don't need to repeat the same condition 3 times

File: src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.html [hunk 2]
```diff
@@ -686,6 +682,8 @@ <h4>{{ "label.who_contacting" | translate }}</h4>
         </mat-expansion-panel>
         <app-ticket-request [disputeInfo]="lastUpdatedDispute"
           *ngIf="lastUpdatedDispute.emailAddressVerified === true || !(lastUpdatedDispute.emailAddress)"></app-ticket-request>
+        <app-upload [disputeInfo]="lastUpdatedDispute"
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.html:685]: Same as above

**Review by karahanb — APPROVED** [review]
LGTM.

File: src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.html (hunk 1 above)
Comment (saikrishnametpalli-nttdata) [reply src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.html]: Done

File: src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.html (hunk 2 above)
Comment (saikrishnametpalli-nttdata) [reply src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.html]: Done

**Review by Sharkww — APPROVED** [review]

**Pull request merged**